src/main/java/mateusz/michal/chat/Controller/RegistrationController.java

```diff
@@ -17,7 +18,7 @@
     RegistrationService registrationService;
 
     @PostMapping(value = "/register",consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<JsonRespond> createNewUser(@RequestBody UserDTO userDto){
+    public ResponseEntity<IJsonResponse> createNewUser(@RequestBody UserDTO userDto){
```

Comment:
To chyba znaczy że możesz usunąć `import mateusz.michal.chat.Model.JsonRespond;`?